### PR TITLE
delta lake example

### PR DESCRIPTION
**Pull request by rajarshighosal**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/data-on-eks/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

Delta Lake Example

### Motivation
Delta Lake example 
1. Upload initial csv and store in delta format
2. Create table catalog and symlink manifest to make file queryable in Athena.
3. Upload updated csv file and merge with original delta table.
4. Query updated file in Athena.

### More

- [x] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [x] Mandatory for new blueprints. Yes, I have added a example to support my blueprint PR
- [x] Mandatory for new blueprints. Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: analytics/terraform/emr-eks-karpenter/examples/nvme-ssd/deltalake/scripts/delta-merge.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      "firstName": "emp_updated.firstName",
+      "lastName": "emp_updated.lastName",
+      "ssn": "emp_updated.ssn",
+      "salary": "emp_updated.salary",
+      "checksum":"emp_updated.checksum"
+    }
+  ) \
+  .execute()
+ 
+
```
Comment (vara-bonthu) [inline analytics/terraform/emr-eks-karpenter/examples/nvme-ssd/deltalake/scripts/delta-merge.py:84]: nit:remove extra lines

File: analytics/terraform/emr-eks-karpenter/examples/nvme-ssd/deltalake/scripts/delta-merge.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+emp_schema = StructType(
+      [ StructField("id",IntegerType(),True), 
+       StructField("firstname",StringType(),True), 
+       StructField("lastname",StringType(),True), 
+       StructField("ssn", StringType(), True),
+       StructField("salary", IntegerType(), True)])
+
+
+#initial file load
+
```
Comment (vara-bonthu) [inline analytics/terraform/emr-eks-karpenter/examples/nvme-ssd/deltalake/scripts/delta-merge.py:30]: nit:remove extra lines in the script.

**Review by vara-bonthu — COMMENTED** [review]
Thanks for this example @rajarshighosal  👍🏼  One minor comment on extra lines.
Could you also update emr karpenter website doc and explain the execution, script functionality and show the output values.

**Comment by rajarshighosal** [comment]
[](url)
Screenshot 2023-04-09 at 5 06 17 PM

@vara-bonthu - should i update the documentation at the bottom of this page for delta-lake or create a child leaf below EMR on EKS with Karpentar

**Comment by vara-bonthu** [comment]
Yes that's correct. Add this example doc with react collapse feature. You need to install node locally to test the docs. Let me know if you need any help.

**Comment by rajarshighosal** [comment]
Screenshot 2023-04-10 at 10 18 14 AM

I added the delta lake md file as shown above but cant make it collapsible.

Here is the code I have for the page:

Screenshot 2023-04-10 at 10 20 21 AM

@vara-bonthu  need your help here

**Review by vara-bonthu — APPROVED** [review]

**Comment by vara-bonthu** [comment]
@rajarshighosal Could you resolve the conflicts?  
We need to create a new policy for Glue table access and add that as a additional policy to EMR on EKS module under emr-eks.tf file. You can do this in the next PR.

**Comment by rajarshighosal** [comment]
@vara-bonthu - I have resolved the conflicts.
Also, I have directly the glue managed service role  to EMR on EKS module in the emr-eks.tf file.

**Pull request merged**